flocker/node/agents/functional/test_ebs.py

```diff
@@ -182,6 +188,132 @@ def test_next_device_in_use(self):
                                        {u"/dev/sdf"})
         self.assertEqual(result, u"/dev/sdg")
 
+    def test_create_volume_gold_profile(self):
+        """
+        Requesting ``gold`` profile during volume creation honors
+        ``gold`` attributes.
+        """
+        self._assert_create_volume_with_mandatory_profile(
+            MandatoryProfiles.GOLD)
+
+    @capture_logging(assertHasMessage, CREATE_VOLUME_FAILURE)
+    def test_create_too_small_volume_gold_profile(self, logger):
+        """
+        Too small volume for ``gold`` attributes creates volume with
+        ``default`` profile.
+        """
+        self._assert_create_volume_with_mandatory_profile(
+            MandatoryProfiles.GOLD, created_profile=MandatoryProfiles.DEFAULT,
+            size_GiB=1)
+
+    def test_create_largest_volume_gold_profile(self):
+        """
+        Largest volume with ``gold`` profile.
+        """
+        raise SkipTest(u'This test is expensive (money).')
```

Comment:
I assume we delete volumes as soon as test is over; is it really that expensive?